src/examples/live-data-basic/example-live-data-basic.component.ts

```diff
@@ -19,36 +21,36 @@ export class ExampleLiveDataBasicComponent implements OnDestroy {
 	title = EXAMPLE_TAGS[1];
 
 	rows: Quote[];
-	timeoutId: any = null;
+
+	private destroy$: Subject<void> = new Subject();
 
 	constructor(private dataService: DataService, private cd: ChangeDetectorRef) {
 		this.dataService.getQuotes().subscribe(quotes => {
 			this.rows = quotes;
-			this.update(true);
+			this.cd.detectChanges();
+
+			interval(300).pipe(
```

Comment:
can we make it more complex? so this interval is not static? and for instance, 2 rows can be updated in one iteration.